src/components/Tag/Tag.tsx

```diff
@@ -34,7 +34,7 @@ export interface ITagProps
 	isRemovable: boolean;
 
 	/** Style variations of the `Tag`. */
-	kind?: 'primary' | 'success' | 'warning' | 'danger' | 'info' | 'default';
```

Comment:
Isn't this technically a breaking change?

Reply:
We never included a color for primary in the css. It's the same color as info. I can keep it there if you'd like and give it the same color blue. Would you prefer that?

Reply:
@jondlm I've added primary back as an option as you suggested and gave it style in the css.